server/src/main/java/com/flipkart/varadhi/components/ComponentKind.java

```diff
@@ -0,0 +1,12 @@
+package com.flipkart.varadhi.components;
+
+public enum ComponentKind {
```

Comment:
not sure about this yet. I am not sure how the configs are going to split; if the components are going to be deployed separately and how the configs are going to merge, if all components are hosted in a standalone pod.

Reply:
yeah.. current plan is to evolve component specific config under its namespace. As of now, no config for controller yet.